### ISSUE TITLE
Using class of type in test/suite parameters causes TypeError error during json.dumps() when creating HTML report

### ISSUE THREAD

**Issue by Rdvp1514** [ArturSpirin/test_junkie#30]
1. Python version `Python 3.6.0`
2. Test Junkie version `0.7a8` 
3. Platform aka *MAC OS Catalina 10.15.1* 
### Expected behavior
created the html Report

### Actual behavior
```
Traceback (most recent call last):
  File "/scr/test/Runner.py", line 10, in <module>
    aggregator = runner.run()
  File "/venv/lib/python3.6/site-packages/test_junkie/runner.py", line 188, in run
    reporter.generate_html_report(self.__settings.html_report)
  File "/venv/lib/python3.6/site-packages/test_junkie/reporter/html_reporter.py", line 110, in generate_html_report
    html = html.format(body=body, database_lol=json.dumps(table_data["database_lol"]))
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/json/__init__.py", line 231, in dumps
    return _default_encoder.encode(obj)
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/json/encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/json/encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/json/encoder.py", line 180, in default
    o.__class__.__name__)
TypeError: Object of type 'type' is not JSON serializable
```

---
**Nota**
You have to deleted in the library html_report.py the json.dumps in the line 110
path : test_junkie/reporter/html_reporter.py

**Comment by ArturSpirin** [comment]
@Rdvp1514  Hello, do you have a code snippet that can repro this issue?

**Comment by Rdvp1514** [comment]
Hello, sure! this the code ```html = html.format(body=body, database_lol=(table_data["database_lol"]))``` this is the code that I fixed

**Comment by Rdvp1514** [comment]
Hellow how are you? this the code that you have ```     body = "{}</div>{}</div>{}".format(row_one_html, row_two_html, ReportTemplate.get_donation_options())
        html = html.format(body=body, database_lol=json.dumps(table_data["database_lol"]))```
and this is the code that I fixe..

```        body = "{}</div>{}</div>{}".format(row_one_html, row_two_html, ReportTemplate.get_donation_options())
        html = html.format(body=body, database_lol=(table_data["database_lol"]))```

**Comment by fabiand93** [comment]
I seems that table_data are not always created

**Comment by ArturSpirin** [comment]
I was able to reproduce this issue with the following code:

```python
from test_junkie.decorators import Suite, test


@Suite()
class ExampleSuite:

    @test(parameters=[str, int, object])
    def archive_product(self, parameter):
        pass


if "__main__" == __name__:
    from test_junkie.runner import Runner
    runner = Runner([ExampleSuite], html_report="E:\\Development\\test_junkie\\playground\\report.html")
    runner.run()
```

I understand what the problem is now, I'll push a fix soon.
Thanks for reporting @Rdvp1514!

**Comment by ArturSpirin** [comment]
@Rdvp1514  This issue is now fixed in version 0.8a1. Please `pip install test-junkie --upgrade` to get the fix.
